pytorch_lightning/trainer/supporters.py

```diff
@@ -177,3 +178,94 @@ def to_disk(self) -> None:
             # Write predictions for current file to disk
             with fs.open(filepath, "wb") as fp:
                 torch.save(outputs, fp)
+
+
+class AverageValueMeter:
```

Comment:
Can you make a reference to torchnet ?

Reply:
This has been adressed I think, but github doesn't seem to notice.